tox.ini

```diff
@@ -18,6 +19,11 @@ deps =   -r{toxinidir}/requirements.txt
 [testenv:venv]
 commands = {posargs}
 
+[testenv:functional]
+isolated_build = True
+commands =
+        pytest --cov=esileapclient {posargs:esileapclient/tests/functional}
```

Comment:
Quick comment: I think we can leave the coverage flag out here, as I don't know that coverage reports are all that useful for functional tests.

Reply:
Makes sense. I honestly didn't really know what it did, and since it's redundant I removed it.